modules/creating-manifest-file-customized-br-ex-bridge.adoc

```diff
@@ -81,14 +85,15 @@ interfaces:
   ipv4:
     enabled: false
     dhcp: false
+    auto-route-metric: 48 <5>
```

Comment:
also move this down
this goes after line 104, on the `br-ex` `ovs-interface`, when `dhcp: true`